src/services/user/registration.js

```diff
@@ -125,80 +142,102 @@ const registerUser = function register(data, params, app) {
 				}
 				return Promise.reject('Ungültiger Link');
 			});
-	}).then(() => populateUser(app, data)
-		.then((response) => {
-			user = response.user;
-			oldUser = response.oldUser;
-		})).then(() => {
-		const consentSkipCondition = Configuration.get('SKIP_CONDITIONS_CONSENT');
-		if ((user.roles || []).includes('student')) {
-			// wrong birthday object?
-			if (user.birthday instanceof Date && Number.isNaN(user.birthday)) {
-				return Promise.reject(new errors.BadRequest(
-					'Fehler bei der Erkennung des ausgewählten Geburtstages.'
-						+ ' Bitte lade die Seite neu und starte erneut.',
-				));
-			}
-			// wrong age?
-			const age = getAge(user.birthday);
-			if (data.parent_email && age >= CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS) {
-				return Promise.reject(new errors.BadRequest(
-					`Schüleralter: ${age} Im Elternregistrierungs-Prozess darf der Schüler`
-						+ `nicht ${CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS} Jahre oder älter sein.`,
-				));
-			} if (!data.parent_email && age < CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS
-				&& !consentSkipCondition.includes('student')) {
-				return Promise.reject(new errors.BadRequest(
-					`Schüleralter: ${age} Im Schülerregistrierungs-Prozess darf der Schüler`
-						+ ` nicht jünger als ${CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS} Jahre sein.`,
-				));
+		})
+		.then(() =>
+			populateUser(app, data).then((response) => {
+				user = response.user;
+				oldUser = response.oldUser;
+			})
+		)
+		.then(() => {
+			const consentSkipCondition = Configuration.get('SKIP_CONDITIONS_CONSENT');
+			if ((user.roles || []).includes('student')) {
+				// wrong birthday object?
+				if (user.birthday instanceof Date && Number.isNaN(user.birthday)) {
+					return Promise.reject(
+						new errors.BadRequest(
+							'Fehler bei der Erkennung des ausgewählten Geburtstages.' + ' Bitte lade die Seite neu und starte erneut.'
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)